docs/content/modeling/contextual-time-based-authorization.mdx

```diff
@@ -802,13 +875,21 @@ In that case, we can extend the model like so:
 <AuthzModelSnippetViewer
   configuration={{
     type_definitions: [
+      {
+        type: 'user',
+      },
       {
         type: 'bank',
         relations: {
           admin: {
             this: {},
           },
         },
+        metadata: {
+          relations: {
+            admin: { directly_related_user_types: [{type: 'user'}, {type: 'branch', relation: 'account_manager'}] },
```

Comment:
```suggestion
            admin: { directly_related_user_types: [{type: 'user'}] },
```

Is there really a situation where a branch's manager would be the admin on all the bank? That seems weird